### ISSUE TITLE
Incorrectly removing code

### ISSUE THREAD

**Issue by azdavis** [rollup/rollup#901]
bool.js:

```js
export default true
```

index.js:

```js
import bool from "./bool"

const hs = document.documentElement.style

if (bool) {
    hs.color = "#222"
}

console.log("hey")
```

command:

```
rollup index.js
```

output:

```js
console.log("hey")
```

😢

**Comment by snuggs** [comment]
@azdavis  The real mystery is why does it work with a log within the predicate block? ¯\_(ツ)_/¯
![capture d ecran 2016-09-06 a 04 54 48](https://cloud.githubusercontent.com/assets/38223/18267662/2fdd2a60-73ee-11e6-9cf8-1aead639df91.png)

**Comment by Rich-Harris** [comment]
Thanks – this will be fixed by #902

**Comment by snuggs** [comment]
@Rich-Harris you ALWAYS come through bruh!
